study-builder/studies/rarex/quality-of-life-child.conf

```diff
@@ -970,12 +970,75 @@
                     }
                   ]
                 }
+              },
+              {
+                "ruleType": "REQUIRED",
+                "allowSave": true,
+                "hintTemplate": {
+                  "templateType": "TEXT",
+                  "templateText": "$req_hint",
+                  "variables": [
+                    {
+                      "name": "req_hint",
+                      "translations": [
+                        {
+                          "language": "en",
+                          "text": ${i18n.en.required_field_hint}
+                        }
+                      ]
+                    }
+                  ]
+                }
               }
             ]
           },
           "blockType": "QUESTION",
           "blockGuid": null,
-          "shownExpr": "true"
+          "shownExpr": """!user.studies["rarex"].forms["QUALITY_OF_LIFE"].questions["PAIN_INTENSITY_CHILD_PREFERENCE"].answers.hasOption("PREFER_NO_TO_ANSWER")"""
```

Comment:
This is the child variant of this survey right? Shouldn't this be `CHILD_QUALITY_OF_LIFE`?

Reply:
@YuriyPavlovBI :
Are you going to take care of this change to change the stableID to CHILD_QUALITY_OF_LIFE ?

Reply:
Sure